desktop/apps/article2/components/FollowButton.js

```diff
@@ -0,0 +1,31 @@
+import Artist from 'desktop/models/artist.coffee'
+import CurrentUser from 'desktop/models/current_user.coffee'
+import { Following, FollowButton } from 'desktop/components/follow_button/index.coffee'
+import { data as sd } from 'sharify'
+import { map } from 'lodash'
+
+export const setupFollows = () => {
+  if (CurrentUser.orNull()) {
+    return new Following(null, {kind: 'artist'})
+  }
+}
+
+export const setupFollowButtons = (following) => {
+  const artists = map($('.artist-follow'), artist => {
+    return artist.getAttribute('data-id')
+  })
+  map(artists, artist => {
```

Comment:
Don't need map / reduce / filter, etc -- all native methods and we're using babel-polyfill, so cross-browser isn't a concern: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array